library/ziti_ctrl.c

```diff
@@ -33,6 +33,9 @@ const char *const PC_PROCESS_MULTI_TYPE = "PROCESS_MULTI";
 const char *const PC_MAC_TYPE = "MAC";
 const char *const PC_ENDPOINT_STATE_TYPE = "ENDPOINT_STATE";
 
+const char *const ERROR_CODE_UNAUTHORIZED = "UNAUTHORIZED";
+const char *const ERROR_MSG_NO_API_SESSION_TOKEN = "no api session token set for ziti_controller";
```

Comment:
Wasn't sure what kind of `ziti_error` to raise. These are used throughout the defensive API session token checks in all requests that require authentication.